common/winobjc.versioning.common.targets

```diff
@@ -92,12 +94,15 @@
 
       <PackageAssemblyVersion>$(PackageVersion_Major).$(PackageVersion_Minor).$(PackageAssemblyVersion_Build).$(PackageAssemblyVersion_Revision)</PackageAssemblyVersion>
       <PackageAssemblyVersionTrimmed>$([System.UInt64]::Parse($(PackageVersion_Major)).ToString()).$([System.UInt64]::Parse($(PackageVersion_Minor)).ToString()).$([System.UInt64]::Parse($(PackageAssemblyVersion_Build)).ToString()).$([System.UInt64]::Parse($(PackageAssemblyVersion_Revision)).ToString())</PackageAssemblyVersionTrimmed>
-      <PackageAssemblyInformationalVersion>$(PackageVersion)+$(GitVersion_FullBuildMetaData)</PackageAssemblyInformationalVersion>
       <PackageAssemblyDescription>https://github.com/Microsoft/WinObjC</PackageAssemblyDescription>
       <PackageAssemblyCompany>Microsoft</PackageAssemblyCompany>
       <PackageAssemblyCopyright>Copyright (C) Microsoft. All rights reserved</PackageAssemblyCopyright>
       <PackageAssemblyProduct>Objective-C for Windows v$(PackageVersion)</PackageAssemblyProduct>
     </PropertyGroup>
+
+    <Message Text="##vso[build.updatebuildnumber]$(PackageVersion)"/>
+    <Message Text="##vso[build.addbuildtag]$(PackageVersion)" Condition="'$(GitVersion_BranchName)' == ''"/>
```

Comment:
Yes it will rename the build. It is currently being called multiple times. 

---
In reply to: [113063928](https://github.com/Microsoft/WinObjC/pull/2606#discussion_r113063928) [](ancestors = 113063928)